src/Adapter/Order/QueryHandler/GetOrderProductsForViewingHandler.php

```diff
@@ -283,28 +283,23 @@ public function handle(GetOrderProductsForViewing $query): OrderProductsForViewi
     private function setProductImageInformation(&$pack_item): void
     {
         if (isset($pack_item['id_product_attribute']) && $pack_item['id_product_attribute']) {
-            $id_image = Db::getInstance()->getValue('
+            $id_image = (int) Db::getInstance()->getValue('
```

Comment:
Don't do that. Only cast $id_image when using `new Image((int) $id_image)`

Reply:
You're right (getValue => false or int).